sphinx/registry.py

```diff
@@ -467,7 +467,9 @@ def get_envversion(self, app: "Sphinx") -> Dict[str, str]:
 
 
 def merge_source_suffix(app: "Sphinx", config: Config) -> None:
-    """Merge source_suffix which specified by user and added by extensions."""
+    """
+    Merge any user-specified source_suffix with any added by extensions.
+    """
```

Comment:
This is also. One-liner is allowed in PEP-8 (see https://www.python.org/dev/peps/pep-0008/#documentation-strings)

Reply:
Again I think this was my attempt to reduce line length (was not sure if there was a limit). Will revert both of these changes in a new commit (hope this is the correct way to resolve).